var/spack/repos/builtin/packages/augustus/package.py

```diff
@@ -30,10 +30,12 @@ class Augustus(MakefilePackage):
        genomic sequences"""
 
     homepage = "http://bioinf.uni-greifswald.de/augustus/"
-    url      = "http://bioinf.uni-greifswald.de/augustus/binaries/augustus-3.3.tar.gz"
+    url      = "http://bioinf.uni-greifswald.de/augustus/binaries/augustus-3.3.1.tar.gz"
```

Comment:
Can you add the following line:
```python
list_url = "http://bioinf.uni-greifswald.de/augustus/binaries/old/"
```
and remove all of the version-specific URLs and see if you can download all of the versions?

Reply:
I've added the line you mentioned and ran **spack checksum augustus** to verify the functionality of the package.

Reply:
`spack checksum` isn't useful, it only proves that you can find new versions. You should try `spack fetch`. Also, you deleted all of the versions. Now the package doesn't have any versions available.

Reply:
Thanks for the information. I've added back the versions I originally had on the 1st commit. Also, I've been working on some other updates for this package, but it's gonna take some time, as I'm trying to test different things.